src/Umbraco.Core/Models/ILanguage.cs

```diff
@@ -55,7 +55,20 @@ public interface ILanguage : IEntity, IRememberBeingDirty
     ///         define fallback strategies when a value does not exist for a requested language.
     ///     </para>
     /// </remarks>
-    [Obsolete("This will be replaced by fallback language ISO code in V13.")]
+    [Obsolete("This will be replaced by fallback language ISO code in V14.")]
     [DataMember]
     int? FallbackLanguageId { get; set; }
+
+
+    /// <summary>
+    ///     Gets or sets the ISO code of a fallback language.
+    /// </summary>
+    /// <remarks>
+    ///     <para>
+    ///         The fallback language can be used in multi-lingual scenarios, to help
+    ///         define fallback strategies when a value does not exist for a requested language.
+    ///     </para>
+    /// </remarks>
+    [DataMember]
+    string? FallbackIsoCode { get; set; }
```

Comment:
Same thing with breaking changes here, since this is nullable though I guess we can just default it to return null.